pants-plugins/internal_plugins/releases/register.py

```diff
@@ -131,10 +131,9 @@ async def check_default_tools(
                     SessionValues(
                         {
                             OptionsBootstrapper: OptionsBootstrapper(
-                                tuple(),
-                                ("./pants",),
+                                FrozenDict(),
```

Comment:
This is because I switched the type of `env` to be a mapping instead of the less obvious list of pairs.